Dockerfile

```diff
@@ -45,6 +45,7 @@ ENV SESSION_WARN=1500
 ENV SESSION_EXPIRE=1800
 ENV STATIC_ROOT="${TETHYS_PERSIST}/static"
 ENV WORKSPACE_ROOT="${TETHYS_PERSIST}/workspaces"
+ENV MEDIA_ROOT="${WORKSPACE_ROOT}"
 ENV QUOTA_HANDLERS="\"[]\""
```

Comment:
Do we also need to add a `MEDIA_URL` env var to the `Dockerfile`?

Reply:
Yes, I think we do. I have added that.